src/fidesops/ops/models/manual_webhook.py

```diff
@@ -23,3 +27,45 @@ class AccessManualWebhook(Base):
     )
 
     fields = Column(MutableList.as_mutable(JSONB), nullable=False)
+
+    @property
+    def fields_schema(self) -> BaseSchema:
+        """Build a dynamic Pydantic schema from fields defined on this webhook"""
```

Comment:
this is really cool to see how you've build this dynamic schema 🙌

Reply:
ah thanks!